Analysis/DEEP2_R23_O32/Binning/N_Bins_Grid_Analysis.py

```diff
@@ -73,71 +84,91 @@ def n_times_binned(fitspath, dataset, pdf_pages, outfile, n_split, R23, O32, SNR
         # or something to that effect if so desired.
         # @{
         n_subbins = np.int(np.floor(float(len(sortO32))/n_split))
-        # We have to take the length of the range of n_split because n_split is of type int
-        subbin_arr = np.ones(n_split, dtype=int)*n_subbins
+        subbin_arr = np.ones(n_split, dtype=int) * n_subbins
         n_remainder = len(sortO32) - n_subbins*n_split
         backwardsIdx = -1
-
+        
```

Comment:
```suggestion
```

Reply:
Should remove the tab without removing extra line.

Reply:
fixed